src/App.tsx

```diff
@@ -30,46 +42,143 @@ const App: FC = () => {
   return (
     <authContext.Provider value={{ user, setUser }}>
       <div className="App">
-        <Layout className="layout">
-          <Header style={{ backgroundColor: '#e6f4ff' }}>
-            <Menu
-              theme="dark"
-              mode="horizontal"
-              defaultSelectedKeys={['2']}
-            ></Menu>
-            <Row style={{ justifyContent: 'space-between' }}>
-              <Col>
-                {user && (
-                  <div>
-                    <Avatar
+        <Layout
+          className="layout"
+          style={{
+            minHeight: '100vh',
+            background: 'linear-gradient(to bottom, #434343, #0B6FB8)',
+          }}
+        >
+          {!user ? (
+            <Login /> // TODO: Style page which shows that you aren't logged in
```

Comment:
Do you know if this TODO is still necessary? Can you remove it if not? 😄

Reply:
It should be removed because it is handled. Thanks!